Makefile

```diff
@@ -1,71 +1,4 @@
-PROG=rke2
-GOLANGCI_VERSION=v1.27.0
-REPO ?= rancher
-IMAGE=${REPO}/rke2-runtime
-K3S_PKG=github.com/rancher/k3s
-RKE2_PKG=github.com/rancher/rke2
-
-ifneq "$(strip $(shell command -v go 2>/dev/null))" ""
-	GOOS ?= $(shell go env GOOS)
-	GOARCH ?= $(shell go env GOARCH)
-else
-	ifeq ($(GOOS),)
-		# approximate GOOS for the platform if we don't have Go and GOOS isn't
-		# set. We leave GOARCH unset, so that may need to be fixed.
-		ifeq ($(OS),Windows_NT)
-			GOOS = windows
-		else
-			UNAME_S := $(shell uname -s)
-			ifeq ($(UNAME_S),Linux)
-				GOOS = linux
-			endif
-			ifeq ($(UNAME_S),Darwin)
-				GOOS = darwin
-			endif
-			ifeq ($(UNAME_S),FreeBSD)
-				GOOS = freebsd
-			endif
-		endif
-	else
-		GOOS ?= $$GOOS
-		GOARCH ?= $$GOARCH
-	endif
-endif
-
-ifndef GODEBUG
-	EXTRA_LDFLAGS += -s -w
-	DEBUG_GO_GCFLAGS :=
-	DEBUG_TAGS :=
-else
-	DEBUG_GO_GCFLAGS := -gcflags=all="-N -l"
-endif
-
-ifdef DRONE_TAG
-	VERSION=$(shell echo ${DRONE_TAG} | sed -e 's/+/-/g')
-else
-	VERSION=dev
-endif
-ifeq (${GOARCH}, "arm64")
-		KUBERNETES_VERSION=${VERSION}
-else
-		KUBERNETES_VERSION=${VERSION}-${GOARCH}
-endif
-
-REVISION=$(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .dirty; fi)
-RELEASE=${PROG}.${GOOS}-${GOARCH}
-
-
-BUILDTAGS     = netgo osusergo
-GO_BUILDTAGS += ${BUILDTAGS}
-GO_BUILDTAGS ?= no_embedded_executor
-GO_BUILDTAGS += ${DEBUG_TAGS}
-GO_TAGS=$(if $(GO_BUILDTAGS),-tags "$(GO_BUILDTAGS)",)
-GO_LDFLAGS=-ldflags '-extldflags "-static"                        \
-					 -X $(RKE2_PKG)/pkg/images.KubernetesVersion=$(KUBERNETES_VERSION)    \
-					 -X $(K3S_PKG)/pkg/version.Program=$(PROG)    \
-					 -X $(K3S_PKG)/pkg/version.Version=$(VERSION) \
-					 -X $(K3S_PKG)/pkg/version.Revision=$(REVISION) $(EXTRA_LDFLAGS)'
-
+TARGETS := $(shell ls scripts | grep -v \\.sh)
```

Comment:
I prefer have explicit targets with dependencies and the help comment to `make help` will work.

Reply:
Basically, I don't want dependencies in scripts, but I do want proper dependencies.  So I should just be able to run `make package` and it does all it needs to.  `make dev-shell`.  Right now `dev-shell` only works if I run the images stuff first myself.

Reply:
got it, will fix